### PR TITLE
Feat/onepanelio.core 348.system.updates

### PR DESCRIPTION
**Pull request by Vafilor**
**What this PR does**:

Code cleanup and update CreateWorkflowExecution to have a WorkflowTemplate as a parameter

**Which issue(s) this PR fixes**:

Fixes onepanelio/core#

**Special notes for your reviewer**:

### CODE REVIEW

File: sqlutil/sqlutil.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
+package sqlutil
```
Comment (rushtehrani) [inline sqlutil/sqlutil.go:1]: Should this be in the `util` directory? `util/sql/sql.go`

Comment (Vafilor) [reply sqlutil/sqlutil.go:1]: Yes, I'll move it.

File: pkg/cron_workflow_types.go [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+		workflowParameter := ParameterFromMap(paramMap)
+
+		parameters = append(parameters, *workflowParameter)
+	}
+
+	return parameters, nil
+}
+
+func (cw *CronWorkflow) GetParametersFromWorkflowSpecJson() ([]byte, error) {
```
Comment (rushtehrani) [inline pkg/cron_workflow_types.go:67]: `GetParametersFromWorkflowSpecJSON()`?

**Comment by rushtehrani** [comment]
Getting this error when selecting a Workspace template in Workspace creation form:

```bash
ERRO[0466] finished unary call with code Unknown         error="no nodePoolOptions in config" grpc.code=Unknown grpc.method=GetWorkspaceTemplate grpc.service=api.WorkspaceTemplateService grpc.start_time="2020-06-19T21:55:02-07:00" grpc.time_ms=243.939 span.kind=server system=grpc
```

**Comment by rushtehrani** [comment]
@Vafilor let's discuss this PR whenever you're back on it, see [my comments in this draft PR](https://github.com/onepanelio/core/pull/364)

**Pull request closed without merging**